src/emulator/downloadableEmulators.ts

```diff
@@ -40,14 +40,14 @@ const DownloadDetails: { [s in DownloadableEmulators]: EmulatorDownloadDetails }
     },
   },
   firestore: {
-    downloadPath: path.join(CACHE_DIR, "cloud-firestore-emulator-v1.11.5.jar"),
-    version: "1.11.5",
+    downloadPath: path.join(CACHE_DIR, "cloud-firestore-emulator-v1.11.6.jar"),
+    version: "1.11.6",
     opts: {
       cacheDir: CACHE_DIR,
       remoteUrl:
-        "https://storage.googleapis.com/firebase-preview-drop/emulator/cloud-firestore-emulator-v1.11.5.jar",
-      expectedSize: 63287761,
-      expectedChecksum: "badd7007623ce51761d7a8ec2be42f8f",
+        "https://storage.googleapis.com/firebase-preview-drop/emulator/cloud-firestore-emulator-v1.11.6.jar",
+      expectedSize: 91311475,
```

Comment:
~30MB size increase? I wonder what is going on here...

Reply:
Resolved offline - command is missing "-c opt"; Yuchen fixed the build script :)